alphagov/paas-cf
# Bump stemcell and cflinux2

What
----

Covering the:

- [USN-3693-1](https://www.cloudfoundry.org/blog/usn-3693-1/)
- [USN-3689-1](https://www.cloudfoundry.org/blog/usn-3689-1/)
- [USN-3696-2](https://www.cloudfoundry.org/blog/usn-3696-2/)
- [USN-3692-1](https://www.cloudfoundry.org/blog/usn-3692-1/)

How to review
-------------

- Make sure the CVEs are covered
- Run through your pipeline
